editoast/src/views/timetable/stdcm.rs

```diff
@@ -107,6 +109,19 @@ pub struct STDCMRequestPayload {
     #[serde(default)]
     #[schema(value_type = Option<String>, example = json!(["5%", "2min/100km"]))]
     margin: Option<MarginValue>,
+    /// Total tonnage of the consist in tons
+    tonnage: Option<f64>,
```

Comment:
Even if the user interface request tons. We should keep use consistent unit over the API.

```suggestion
    /// Total mass of the consist in kg
    mass: Option<f64>,
```

Reply:
On `front`, `tonnage` has been used. We should be consistent. See https://github.com/OpenRailAssociation/osrd/pull/8940#discussion_r1784422643.

Reply:
> About "tonnage" vs "mass", do we want something different displayed to the user that what we use internally in variable names and identifiers? If yes, is there a compelling reason to do so?

The frontend is not the only user of our API. I prefer a consistent API, `kg` is what we're using for rolling stocks.